tests/kernel/mellanox_ofed.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Mellanox OFED package installation
+# This package is not ready for SLE15 yet, there is an available
+# package for SLE12-SP3 which is able to build the needed RPMs
+# for the current OS version.
```

Comment:
not clear for me what is "current OS version" ? any version in which this test would run ?

Reply:
ok, will modify it to SLE15